metacat-client/src/main/java/com/netflix/metacat/client/api/MetacatV1.java

```diff
@@ -236,7 +237,10 @@ DatabaseDto getDatabase(
             String databaseName,
         @DefaultValue("true")
         @QueryParam("includeUserMetadata")
-            Boolean includeUserMetadata
+            Boolean includeUserMetadata,
+        @DefaultValue("true")
+        @QueryParam("includeTableNames")
+            Boolean includeTableNames
```

Comment:
Since you're using a default value do you still need to autobox it into a `Boolean`? Can you just use the native boolean type and save the small bit over overhead?

Reply:
I kept it consistent with other REST APIs.